digitalfabrik/integreat-app
# IGAPP-1122 Hide Chat on Location Route

This pull request belongs to an issue on our [bugtracker](https://issues.integreat-app.de/).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword **IGAPP**.

To test:
1. Go to http://localhost:9000/muenchen/de/locations
2. See that there is no chatbot icon
3. http://localhost:9000/muenchen/de/news/local
4. See that there is a chatbot icon
